src/main/resources/api/service_openapi.yaml

```diff
@@ -459,6 +459,101 @@ components:
         hybridConnectionName:
           type: string
           description: The user-provided name to identify the Relay HybridConnection.
+    AzureContainerInstance:
+      type: object
+      required:
+        - resourceGroup
+        - containerGroupName
+        - containers
+        - operatingSystemTypes
+      properties:
+        resourceGroup:
+          $ref: '#/components/schemas/AzureResourceGroup'
+        containerGroupName:
+          type: string
+          description: The name of the container group
+        containers:
```

Comment:
It would be handy to add the `description` field for each new object definition.